xla/service/llvm_ir/llvm_util.cc

```diff
@@ -50,6 +50,7 @@ limitations under the License.
 #include "llvm/Support/Casting.h"
 #include "llvm/Support/CodeGen.h"
 #include "llvm/Support/raw_ostream.h"
+#include "llvm/TargetParser/Triple.h"
```

Comment:
This needs a corresponding dependency in the BUILD file for the llvm_util target.

Reply:
Done.